src/CollAction/Views/Projects/Create.cshtml

```diff
@@ -8,89 +8,83 @@
+                    </div>
+                    <div class="form-group">
+                        <label asp-for="Goal" class="col-md-4 control-label"></label>
+                        <div class="col-md-10">
+                            <input asp-for="Goal" class="form-control" />
+                            <span asp-validation-for="Goal" class="text-danger" />
+                        </div>
+                    </div>
+                    <div class="form-group">
+                        <label asp-for="CategoryId" class="col-md-4 control-label"></label>
+                        <div class="col-md-10">
+                            @Html.DropDownList("CategoryId", Model.Categories)
+                            <span asp-validation-for="CategoryId" class="text-danger" />
+                        </div>
+                    </div>
+                    <div class="form-group">
+                        <label asp-for="LocationId" class="col-md-4 control-label"></label>
+                        <div class="col-md-10">
+                            @Html.DropDownList("LocationId", Model.Locations)
+                            <span asp-validation-for="LocationId" class="text-danger" />
+                        </div>
+                    </div>
+                    <div class="form-group">
+                        <label asp-for="Target" class="col-md-4 control-label"></label>
+                        <div class="col-md-10">
+                            <input asp-for="Target" class="form-control" />
+                            <span asp-validation-for="Target" class="text-danger" />
+                        </div>
+                    </div>
+                    <div class="form-group">
+                        <label asp-for="End" class="col-md-4 control-label"></label>
+                        <div class="col-md-10">
+                            <input asp-for="End" class="form-control" />
+                            <span asp-validation-for="End" class="text-danger" />
+                        </div>
+                    </div>
+                    <div class="form-group">
+                        <label asp-for="CreatorComments" class="col-md-4 control-label"></label>
+                        <div class="col-md-10">
+                            <input asp-for="CreatorComments" class="form-control" />
+                            <span asp-validation-for="CreatorComments" class="text-danger" />
+                        </div>
+                    </div>
+                    <div class="form-group">
+                        <div class="col-md-10">
+                            <input type="submit" value="Create" class="btn btn-default" />
+                        </div>
+                    </div>
+                </div>
+            </form>
         </div>
     </div>
-</form>
-
-<div>
-    <a asp-action="Index">@Localizer["Back to List"]</a>
-</div>
-
-@section Scripts {
-    @{await Html.RenderPartialAsync("_ValidationScriptsPartial");}
```

Comment:
The server will still validate but it will be a bit heavy loading both jquery and the whole react bundle. There is a way to get both the client side react application as well as client side form validation based on the ASP.net view model pattern but I was going to do that post launch. Thoughts?